packages/orbit-components/src/hooks/useRandomId/index.ts

```diff
@@ -1,14 +1,17 @@
-import { useUIDSeed, useUID } from "react-uid";
+import { useRandomId as useRandomIdContext } from "../../OrbitProvider/RandomId/Provider";
 
 type UseRandomIdSeed = () => (id: string) => string;
 type UseRandomId = () => string;
 
-export const useRandomIdSeed: UseRandomIdSeed = () => {
-  return useUIDSeed();
+export const useRandomId: UseRandomId = () => {
```

Comment:
Let's add a jsDoc deprecation message here and move this hook maybe to deprecated folder 🤔

Reply:
I think it's okay to keep it for now, once we officially don't support React 17, we can clean this up and remove these hooks if they prove to be unnecessary (since it's basically just using `React.useId` anyway).

Reply:
> _We can decide to deprecate it, but what is the alternative? I think for deprecation we need to have a solution that works for both versions of React or officially deprecate all support for React <18, which is not something we are planning to do right now, right?_

The alternative is React.used hook that will replace that, deprecated does not mean that it's not in use, it means that it will be removed and it will be, and not necessarily in the next major version. The deprecation period may vary.

Reply:
I get what you're saying, but I think we should only do that when we officially deprecate React 17 support entirely (which we are not doing yet, right?). I think, for now, the best option is to just add a note on the documentation that this hook should only be used by React <18 users. WDYT?